com.unity.render-pipelines.universal/Documentation~/rendering/deferred-rendering-path.md

```diff
@@ -0,0 +1,422 @@
+    </tr>
+    <tr>
+        <td></td>
+        <td>Deferred rendering (stencil)</td>
+    </tr>
+    <tr>
+        <td>AfterRenderingDeferredLights</td>
+        <td>&#160;</td>
+    </tr>
+    <tr>
+        <td>BeforeRenderingOpaques</td>
+        <td>Opaque Forward-only Materials</td>
+    </tr>
+    <tr>
+        <td>AfterRenderingOpaques</td>
+        <td>&#160;</td>
+        <td>SSAO and blending (optional)</td>
+    </tr>
+    <tr>
+        <td>BeforeRenderingSkybox</td>
+        <td>&#160;</td>
+        <td>&#160;</td>
+    </tr>
+    <tr>
+        <td>AfterRenderingSkybox</td>
+        <td>&#160;</td>
+        <td>&#160;</td>
+    </tr>
+    <tr>
+        <td>BeforeRenderingTransparents</td>
+        <td>&#160;</td>
+        <td>&#160;</td>
+    </tr>
+    <tr>
+        <td>AfterRenderingTransparents</td>
+        <td>&#160;</td>
+        <td>&#160;</td>
+    </tr>
+    <tr>
+        <td>BeforeRenderingPostProcessing</td>
+        <td>&#160;</td>
+        <td>&#160;</td>
+    </tr>
+    <tr>
+        <td>AfterRenderingPostProcessing</td>
+        <td>&#160;</td>
+        <td>&#160;</td>
+    </tr>
+    <tr>
+        <td>AfterRendering</td>
+        <td>&#160;</td>
+        <td>&#160;</td>
+    </tr>
+    <tbody>
+</table>
+
+The following sections describe the Deferred Rendering Path render Passes.
+
+#### Depth, or depth and normal prepass
+
+In the Deferred Rendering Path, in the depth prepass or depth and normal prepass, Unity renders only the Materials that do not support the deferred rendering model. For example, Materials using the Complex Lit shader are such Materials.
+
+In the Deferred Rendering Path, Unity does not use the depth prepass to generate a copy of the depth buffer (this behavior is different in the Forward Rendering Path).
+
+If the Universal Renderer has the SSAO Renderer Feature, Unity executes the depth and normal prepass. SSAO uses the screen-space depth and normal buffers to calculate ambient occlusion.
+
+#### Optional passes: SSAO, SSAO with blending
+
+If the Universal Renderer has the SSAO Renderer Feature, and the **After Opaque** option is disabled (the option is disabled by default), Unity executes the SSAO Pass at the AfterRenderingGbuffer event. The SSAO Renderer Feature calculates the SSAO texture. Unity samples this texture in the Deferred rendering Pass and in the Pass that renders Forward-only Materials.
+
+Using this Pass order, Unity can combine the baked occlusion with the real-time occlusion from the SSAO Renderer Feature, and avoid double-darkening from the baked and the real-time ambient occlusion.
+
+When the After Opaque option is enabled, Unity executes the SSAO and blending Pass at the AfterRenderingOpaques event, after rendering the Forward-only Materials. Unity then executes an extra full screen Pass to overlay the SSAO texture onto the Emissive/GI/Lighting buffer. This causes over-darkening of the areas that receive baked occlusion and real-time occlusion.
```

Comment:
@phi-lira @ellioman Thte depth-normal prepass aways happens when SSAO is enabled, regardless of the After Opaque option. An user recently commented in the forums that we should enable the depth (or depth-normal) prepass all the time in deferred, so that forward-only materials can be primed into the depth buffer as an optimisation. The user also indicated this is what the built-in deferred renderer does.